### PR TITLE
Fix import for LayerDetailView plugin

### PR DESCRIPTION
**Pull request by dsuren1**

### CODE REVIEW

**Review by giohappy — APPROVED** [review]

File: geonode_mapstore_client/client/js/plugins/LayerDetailViewer.jsx [hunk 1]
```diff
@@ -26,7 +26,18 @@ import { hashLocationToHref } from '@js/utils/SearchUtils';
...
+import DetailsLinkedResources from '@js/components/DetailsPanel/DetailsLinkedResources';
+
+const tabComponents = {
+    'attribute-table': DetailsAttributeTable,
+    'linked-resources': DetailsLinkedResources,
+    'locations': DetailsLocations,
+    'assets': DetailsAssets
+};
 
 const Button = tooltip(GNButton);
```
Comment (ridoo) [inline geonode_mapstore_client/client/js/plugins/LayerDetailViewer.jsx:42]: @dsuren1 there's a missing `export`: 
export default tabComponents;

Comment (ridoo) [reply geonode_mapstore_client/client/js/plugins/LayerDetailViewer.jsx:42]: This should fix it:

```js
/*
 * Copyright 2024, GeoSolutions Sas.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree.
 */

import DetailsLocations from '@js/components/DetailsPanel/DetailsLocations';
import DetailsAssets from '@js/components/DetailsPanel/DetailsAssets';
import DetailsAttributeTable from '@js/components/DetailsPanel/DetailsAttributeTable';
import DetailsLinkedResources from '@js/components/DetailsPanel/DetailsLinkedResources';

const tabComponents = {
    'attribute-table': DetailsAttributeTable,
    'linked-resources': DetailsLinkedResources,
    'locations': DetailsLocations,
    'assets': DetailsAssets
};

export default tabComponents;
```

Comment (ridoo) [reply geonode_mapstore_client/client/js/plugins/LayerDetailViewer.jsx:42]: I saw, you moved `tabComponents` back to `LayerDetailViewer` which is ok, too. Just ignore my comments here

**Comment by giohappy** [comment]
@dsuren1 I have merged this but please check what @ridoo is reporting.

**Comment by ridoo** [comment]
@giohappy I am just fine how @dsuren1 resolved the issue.

**Comment by giohappy** [comment]
@dsuren1 it works fine on stable demo but not on development demo. That's odd, can you check what's wrong?

**Comment by ridoo** [comment]
@giohappy yesterday the map creation view trapped me during test while the map view worked as expected. Perhaps the :information_source:  button could be shown there as well (just an idea).

**Pull request merged**